openapi-cli/src/main/java/io/ballerina/openapi/generators/schema/BallerinaTypesGenerator.java

```diff
@@ -118,6 +129,31 @@ public SyntaxTree generateSyntaxTree() throws BallerinaOpenApiException {
         return syntaxTree.modifyWith(modulePartNode);
     }
 
+    private NodeList<ImportDeclarationNode> getImportNodeForHttpModule(NodeList<ImportDeclarationNode> imports) {
```

Comment:
Idea was to move any future import generations(if we will have them) to move inside this function and return the related import list since it is easy to handle in one place, currently, we have only http import, therefore methodName is named with adding http. May I change the return type or method name according to? 🙂

Reply:
Yeah we better change either the method name or the return type :)